jvcl/run/JvRollOut.pas

```diff
@@ -1451,13 +1451,17 @@ procedure TJvCustomRollOut.CheckChildVisibility;
       FChildControlVisibility.Sorted := True;
     end;
 
+    DisableAlign;
+
     for I := 0 to ControlCount - 1 do
       if (Controls[I] is TWinControl) and (TWinControl(Controls[I]).Visible) then
       begin
         FChildControlVisibility.AddObject(Controls[I].Name, Controls[I]);
         if CollapseCtrlsOnButton or (TWinControl(Controls[I]).Top > ButtonHeight) then
           TWinControl(Controls[I]).Visible := False;
       end;
+
+    EnableAlign := true;
```

Comment:
This looks weird, either it's a method like below, or it's a property. Does this even compile?
And I agree with @baka0815, this must be placed inside a `try..finally` block